rca/static_src/sass/components/_hero.scss

```diff
@@ -3,14 +3,28 @@
     overflow: hidden;
 
     &--medium {
-        max-height: 340px;
+        &::before {
+            @include z-index(under);
+            content: '';
+            background-color: $color--black;
+            height: 130px;
+            left: 0;
+            position: absolute;
+            top: 0;
+            width: 100%;
+        }
```

Comment:
Scrolled down on both desktop and mobile:

<img width="1800" alt="Screenshot 2025-04-24 at 7 05 33 PM" src="https://github.com/user-attachments/assets/fb70867f-cee2-4b2c-abdb-fa1827f2f409" />

<img width="450" alt="Screenshot 2025-04-24 at 7 05 40 PM" src="https://github.com/user-attachments/assets/d51f6e67-8ab1-4456-b16f-b0a50810ca10" />